devel/ansible/roles/dev/files/motd

```diff
@@ -8,6 +8,7 @@ bstart:     Start the Bodhi service.
 bstop:      Stop the Bodhi service.
 btest:      Run Bodhi's test suite.
 bteststyle: Run just the flake8 code style tests.
+bfedmsg:    View all fedmsgs Bodhi has emitted.
```

Comment:
Perhaps it is dumb that this is double documented, but there is also a section in the docs that talks about these commands:

https://bodhi.fedoraproject.org/docs/developer_docs.html#quick-tips-about-the-bodhi-vagrant-environment

You think we should delete the double docs, or should we add this to it?